new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Operator.java

```diff
@@ -3,35 +3,78 @@
 
 package oracle.weblogic.kubernetes.actions.impl;
 
+import io.kubernetes.client.openapi.ApiException;
 import oracle.weblogic.kubernetes.actions.impl.primitive.Helm;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
 
-public class Operator {
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class Operator implements LoggedTest {
 
   /**
    * The URL of the Operator's Helm Repository.
    */
   private static String OPERATOR_HELM_REPO_URL = "https://oracle.github.io/weblogic-kubernetes-operator/charts";
+
   /**
    * The name of the Operator's Helm Chart (in the repository).
    */
   private static String OPERATOR_CHART_NAME = "weblogic-operator/weblogic-operator";
 
+  /**
+   * Install WebLogic Kubernetes Operator.
+   *
+   * @param params operator parameters for helm values
+   * @return true if the operator is successfully installed, false otherwise.
+   * @throws ApiException - if Kubernetes client API call fails
+   */
+  public static boolean install(OperatorParams params) throws ApiException {
+
+    String namespace = params.getNamespace();
+    String serviceAccount = params.getServiceAccount();
+
+    // assertions for required parameters
+    assertNotNull(namespace, "Operator namespace cannot be null");
+    assertNotNull(params.getReleaseName(), "Operator release name cannot be null");
+    assertNotEquals("",namespace.trim(), "Operator namespace cannot be empty string");
+    assertNotEquals("",params.getReleaseName().trim(), "Operator release name cannot be empty string");
+
+    //delete the namespace if it exists
+    if (Namespace.exists(namespace)) {
```

Comment:
why delete the namespace? seems a bit extreme

Reply:
what if the namespace already exists? just proceed with it?

Reply:
yeah - the method is meant to install an operator --- if doing so deleted the target namespace, that would be unexpected and surprising behavior.  we should not do that

Reply:
ok, will modify the code accordingly